examples/src/main/java/com/radixdlt/client/examples/RadixWalletExample.java

```diff
@@ -38,28 +37,33 @@ public static void main(String[] args) throws Exception {
 			.getStatusUpdates()
 			.subscribe(System.out::println);
 
-
 		RadixApplicationAPI api = RadixApplicationAPI.create(radixIdentity);
```

Comment:
<nitpicking> Should we capitalize it RadixApplicationApi instead for consistency? </nitpicking>

Reply:
Not a fan of changing signatures at the moment as will probably change it down the line again as architecture gets more defined to something like `RadixApplication`